src/cli/Show.cpp

```diff
@@ -49,6 +49,13 @@ int Show::execute(QStringList arguments)
                                QObject::tr("Key file of the database."),
                                QObject::tr("path"));
     parser.addOption(keyFile);
+    QCommandLineOption fields(QStringList() << "a"
+                                            << "show-attribute",
+                              QObject::tr("Attribute name to extract and output. "
```

Comment:
@cyphar what about `names of the attributes to show`, since `show` is the name of the command.